File: Sprint 03/Atividade 11/1.1/tarefa.py
maior_numero = 0
QUANTIDADE_REPETICOES = 6

for i in range(1, QUANTIDADE_REPETICOES):
	numero_digitado = int(input(f"Digite o {i}º número Natural: "))

	while numero_digitado < 0:
		numero_digitado = int(input(f"Digite o {i}º número Natural: "))

	if numero_digitado > maior_numero:
		maior_numero = numero_digitado

print(f"\nO maior número Natural digitado foi {maior_numero}!")

File: Sprint 03/Atividade 11/1.1/Tarefa.java
import java.util.Scanner;

public class Tarefa {
	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);
		int maiorNumero = 0;
		final int QUANTIDADE_REPETICOES = 5;

		for(int i=1; i<=QUANTIDADE_REPETICOES; i++) {
			int numerosDigitados;
			
			do {
				System.out.print("Digite o " + i + "-esimo numero natural: ");
				numerosDigitados = scan.nextInt();
			}while(numerosDigitados < 0);
			
			if(numerosDigitados > maiorNumero) maiorNumero = numerosDigitados;
		}
		
		System.out.println("O maior numero natural digitado foi " + maiorNumero + "!");
		scan.close();
	}
}
